client/packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -14,86 +14,80 @@ const appVersion = require('../../../../../../package.json').version; // eslint-
 declare const LANG_VERSION: string;
 
 const defaultNS = 'common';
+const minuteInMilliseconds = 60 * 1000;
+const isDevelopment = process.env['NODE_ENV'] === 'development';
+const expirationTime = isDevelopment ? 0 : 7 * 24 * 60 * minuteInMilliseconds; // Cache for 7 days, on rebuild we should get a new language version so we can use a reasonably long cache
 
-type IntlProviderProps = PropsWithChildren<{ isElectron?: boolean }>;
+const languageVersion =
+  typeof LANG_VERSION === 'undefined' ? appVersion : LANG_VERSION;
 
-export const IntlContext = createRegisteredContext<I18nextProviderProps>(
-  'i18nextProvider',
-  { i18n: i18next }
-);
-
-export const IntlProvider: FC<IntlProviderProps> = ({
-  isElectron,
-  children,
-}) => {
-  React.useEffect(() => {
-    if (i18next.isInitialized) return;
-
-    const languageDetector = new LanguageDetector();
-    languageDetector.addDetector(browserLanguageDetector);
-
-    const minuteInMilliseconds = 60 * 1000;
-    const isDevelopment = process.env['NODE_ENV'] === 'development';
-    const expirationTime = isDevelopment
-      ? 0
-      : 7 * 24 * 60 * minuteInMilliseconds; // Cache for 7 days, on rebuild we should get a new language version so we can use a reasonably long cache
+/** Must be initialised outside of react */
+export function initialiseI18n({
```

Comment:
Ah sorry, bit of noise in here - it's all just shuffling. Only change is the addition of this function, to be called by app root, rather than inside useEffect